src/frontend/components/TeacherLiveRawWrapper/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import DashboardVideoLiveRaw from 'components/DashboardVideoLiveRaw';
+import VideoPlayer from 'components/VideoPlayer';
+import { useLiveFeedback } from 'data/stores/useLiveFeedback';
+import { Video } from 'types/tracks';
+
+interface TeacherLiveRawWrapperProps {
+  video: Video;
+}
+
+const TeacherLiveRawWrapper = ({ video }: TeacherLiveRawWrapperProps) => {
+  const isDisplayLiveFeedback = useLiveFeedback()[0];
```

Comment:
I don't like this way to use a hook. Isn't it possible to use something like `  const [isDisplayLiveFeedback, _ ]  = useLiveFeedback();`